public/locales/cy/translations.json

```
@@ -157,22 +157,22 @@
   "mutuals": "Cydfuddiol",
-  "Follow": "Dilyn",
-  "Following": "Yn dilyn",
-  "Unfollow": "Dad-ddilyn",
-  "Blocked": "Wedi'i rwystro",
-  "Block": "Bloc",
+  "follow": "Dilyn",
+  "following": "Yn dilyn",
+  "unfollow": "Dad-ddilyn",
+  "blocked": "Wedi'i rwystro",
+  "block": "Bloc",
   "block": "bloc",
```

Biome (1.9.4)

[error] The key block was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)